neo/Network/RPC/RpcServer.cs

```diff
@@ -265,23 +385,162 @@ protected virtual JObject Process(string method, JArray _params)
+                        }
+                    }
+                case "sendmany":
+                    if (wallet == null)
+                        throw new RpcException(-400, "Access denied");
+                    else
+                    {
+                        JArray to = (JArray)_params[0];
+                        if (to.Count == 0)
+                            throw new RpcException(-32602, "Invalid params");
+                        TransferOutput[] outputs = new TransferOutput[to.Count];
+                        for (int i = 0; i < to.Count; i++)
+                        {
+                            UIntBase asset_id = UIntBase.Parse(to[i]["asset"].AsString());
+                            AssetDescriptor descriptor = new AssetDescriptor(asset_id);
+                            outputs[i] = new TransferOutput
+                            {
+                                AssetId = asset_id,
+                                Value = BigDecimal.Parse(to[i]["value"].AsString(), descriptor.Decimals),
+                                ScriptHash = to[i]["address"].AsString().ToScriptHash()
+                            };
+                            if (outputs[i].Value.Sign <= 0)
+                                throw new RpcException(-32602, "Invalid params");
+                        }
+                        Fixed8 fee = _params.Count >= 2 ? Fixed8.Parse(_params[1].AsString()) : Fixed8.Zero;
+                        if (fee < Fixed8.Zero)
+                            throw new RpcException(-32602, "Invalid params");
+                        UInt160 change_address = _params.Count >= 3 ? _params[2].AsString().ToScriptHash() : null;
+                        Transaction tx = wallet.MakeTransaction(null, outputs, change_address: change_address, fee: fee);
+                        if (tx == null)
+                            throw new RpcException(-300, "Insufficient funds");
+                        ContractParametersContext context = new ContractParametersContext(tx);
+                        wallet.Sign(context);
+                        if (context.Completed)
+                        {
+                            tx.Witnesses = context.GetWitnesses();
+                            wallet.ApplyTransaction(tx);
+                            system.LocalNode.Tell(new LocalNode.Relay { Inventory = tx });
+                            return tx.ToJson();
+                        }
+                        else
+                        {
+                            return context.ToJson();
+                        }
+                    }
                 case "sendrawtransaction":
                     {
                         Transaction tx = Transaction.DeserializeFrom(_params[0].AsString().HexToBytes());
-                        return LocalNode.Relay(tx);
+                        RelayResultReason reason = system.Blockchain.Ask<RelayResultReason>(tx).Result;
+                        return GetRelayResult(reason);
```

Comment:
This changes the type of result in the returned response body and makes the API no longer compatible for clients that expected the result to be a boolean true/false value.

Really a different json key should be used to return this result so that it doesn't break clients.

Reply:
Thanks for fixing this :-)